### DISCUSSION TITLE
Status

### DISCUSSION THREAD

**Discussion by Zoomll** [fengari-lua/fengari-web#64] [General]
Is this being maintained, or is it planned to be? I stumbled across this and think it's super cool. I noticed that fengari and fengari-interop have had more "recent" updates. Will fengari-web need to be caught up or something? Or do you think it's fine where it is?

**Comment by daurnimator** [comment]
> Is this being maintained

Barely/No.


> noticed that fengari and fengari-interop have had more "recent" updates. Will fengari-web need to be caught up or something?

It does.
But to do so needs a huge webpack version bump, which was non-trivial and looks like a few hours work. There's like 20 things higher up my priority list so I don't see it happening in the near future.

**Reply by Zoomll** [reply]
Alright.
